README.md

```diff
@@ -81,6 +81,7 @@ You can deploy this project using any hosting server in the world
 ## 🔭 Future Features <a name="future-features"></a>
 
 - [ ] **Full website**
+- Open this link to visit my portfolio website hosted on server: https://husush23.github.io/husush.github.io/
```

Comment:
- [ ] **Good job on deploying your application 👏 👏** However, It would be better to put this link under the **deployment** section to make your readme more informative and professional 👍